inc/Engine/CriticalPath/RESTWP.php

```diff
@@ -100,6 +100,17 @@ public function check_permissions() {
 		return current_user_can( 'rocket_regenerate_critical_css' );
 	}
 
+	/**
+	 * Clean post cache files on CPCSS generation or deletion.
+	 *
+	 * @since 3.6.1
+	 *
+	 * @param int $item_id ID for this item to get Url for.
+	 */
+	private function clean_post_cache( $item_id ) {
+		rocket_clean_files( $this->get_url( $item_id ) );
```

Comment:
A reason to use `rocket_clean_files()` instead of `rocket_clean_post()` ?

Reply:
Yes there is a reason. `rocket_clean_post()` will clean also Homepage & Post related links (like archives, adjacent posts, parent posts). 
However when the CPCSS for a specific page is regenerated it will affect only that specific page. So only the specific page needs a cache purge. 
Toughts?

Reply:
That makes sense 👍